temporalio/workflow.py

```diff
@@ -82,20 +92,20 @@ def defn(cls: Optional[ClassType] = None, *, name: Optional[str] = None):
     Args:
         cls: The class to decorate.
         name: Name to use for the workflow. Defaults to class ``__name__``.
+        sandboxed: Whether the workflow should run in a sandbox. Default is
+            true.
```

Comment:
Would make sense to refer to more detailed docs about what the sandbox is here.

Reply:
I wanted to, but I am afraid of linking to the README considering the goal would be to have it in real docs (but the README is so much more comprehensive). I have this same problem in `workflow_sandbox/__init__.py` where I just say:

> See the Python SDK documentation for how to use, customize, and work around sandbox issues.